pkgs/data/fonts/monocraft/default.nix

```
@@ -18,16 +18,14 @@ stdenv.mkDerivation {
     (relArtifact "Monocraft-nerd-fonts-patched.ttf" "sha256-QxMp8UwcRjWySNHWoNeX2sX9teZ4+tCFj+DG41azsXw=")
   ];
 
-  sourceRoot = ".";
-  unpackCmd = ''cp "$curSrc" $(basename $curSrc)'';
-
+  dontUnpack = true;
   dontConfigure = true;
   dontBuild = true;
 
   installPhase = ''
     runHook preInstall
-    install -Dm644 -t $out/share/fonts/opentype *.otf
-    install -Dm644 -t $out/share/fonts/truetype *.ttf
+    find $srcs -name '*.otf' | xargs install -Dm644 --target $out/share/fonts/opentype
```

Is `find` really necessary here? If it is, `-exec` could be used to avoid xargs
```suggestion
    find $srcs -name '*.otf' -exec install -Dm644 --target $out/share/fonts/opentype {} +
```

---

Yeah nice suggestion! I think `find` is necessary if we want to keep the same behavior as before of having a folder for the opentype and for the truetype varieties